composer.json

```diff
@@ -94,7 +94,9 @@
         "ci:tests:unit": "@php \"./vendor/bin/phpunit\"",
         "composer:normalize": "@php \"./.phive/composer-normalize\"",
         "php:fix": "@php \"./.phive/php-cs-fixer\" --config=config/php-cs-fixer.php fix config/ src/ tests/",
-        "php:version": "@php -v | grep -Po 'PHP\\s++\\K(?:\\d++\\.)*+\\d++(?:-\\w++)?+'"
+        "php:version": "@php -v | grep -Po 'PHP\\s++\\K(?:\\d++\\.)*+\\d++(?:-\\w++)?+'",
+        "psalm:baseline": "@php \"./.phive/psalm\" --update-baseline",
```

Comment:
We'll need to use `--set-baseline=psalm.baseline.xml` here instead to regenerate the baseline AFAIK.

Reply:
I think we're looking at different functionality - `--set-baseline` adds new issues to the baseline, whereas `--update-baseline` removes issues that have been fixed (but does not add any new issues).  I don't know if they can be combined - or maybe `--set-baseline` also removes fixed issues (if it regenerates it from scratch, it will do by definition), so that's the one we want anyway?

Reply:
Then let's first find out whether `---set-baseline` also removes fixed issues, and then come to an informed decision.

Reply:
Yep, `--set-baseline` covers both bases, so have changed to that.